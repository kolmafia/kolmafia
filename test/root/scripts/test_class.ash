boolean ok;
ok = ($class[Avatar of Sneaky Pete] == $class[Avatar of Sneaky Pete]);
print("comparison: " + to_string(ok));
ok = ($class[Gelatinous Noob].primestat == $stat[Moxie]);
print("primestat: " + to_string(ok));
ok = ($class[Ed] == $class[Ed the Undying]);
print("substring matching: " + ok);
ok = ("astral spirit".to_class() == $class[none]);
print("no astral spirit: " + ok);
