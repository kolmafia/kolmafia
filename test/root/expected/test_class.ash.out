Changing "Ed" to "Ed the Undying" would get rid of this message. (test_class.ash, line 6, char 14 to char 16)
comparison: true
primestat: true
substring matching: true
no astral spirit: true
